src/SFA.DAS.EAS.Portal.UnitTests/SFA.DAS.EAS.Portal.UnitTests.csproj

```diff
@@ -9,9 +9,11 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.6.0" />
+    <PackageReference Include="Moq" Version="4.10.1" />
     <PackageReference Include="nunit" Version="3.11.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
```

Comment:
Have we rolled back the version of this package for a reason?

Reply:
No. It just happened.

Reply:
👍